### PR TITLE
Create ui-c8y-1019-18-0-dashboard-custom-gainsight-events-5318-graft-release-cd-6007.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by jdre

Ticket: [MTM-57584](https://cumulocity.atlassian.net/browse/MTM-57584)
Original PR: [6007](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6007)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1019-18-0-dashboard-custom-gainsight-events-5318-graft-release-cd-6007.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Dashboard custom gainsight events. (#5318) [GRAFT][release/cd] (#6007)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1019-18-0-dashboard-custom-gainsight-events-5318-graft-release-cd-6007.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Add custom Gainsight events for dashboard
```

File: content/change-logs/application-enablement/ui-c8y-1019-18-0-dashboard-custom-gainsight-events-5318-graft-release-cd-6007.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-57584
+version: 1019.18.0
+---
+Dashboard custom gainsight events. (#5318) [GRAFT][release/cd] (#6007)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1019-18-0-dashboard-custom-gainsight-events-5318-graft-release-cd-6007.md:17]: The Claude AI suggested to use the following body:
```suggestion
To improve the analytics capabilities of the Cumulocity IoT platform, custom Gainsight events have been added for the dashboard. Previously, only a limited set of predefined events were available which restricted the insights that could be gained from user interactions with the dashboard. With this change, additional custom events are now being tracked, such as button clicks, widget loads and configuration changes. This allows for a much more detailed understanding of how users are interacting with the dashboard and enables better optimizations and tailoring of the user experience based on the captured usage patterns.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Comment by jdreSAG** [comment]
redundant PR

**Pull request closed without merging**